google/cloud/internal/base64_transforms.h

```diff
@@ -36,6 +37,9 @@ class Base64Encoder {
     buf_[len_++] = c;
     if (len_ == buf_.size()) Flush();
   }
+  void PushBack(absl::string_view string) {
+    for (char const c : string) PushBack(c);
+  }
```

Comment:
Notice that you've enshrined a `signed char` (in all probabilities) to `unsigned char` conversion here.  Base64 is intended for "binary data", which means `absl::string_view` was probably the wrong representation in the first place, and why raw `char` was never mentioned here before.